engine/render/src/render/program_utils.cpp

```diff
@@ -360,7 +364,7 @@ namespace dmRender
             }
             case dmRenderDDF::MaterialDesc::CONSTANT_TYPE_VIEWPROJ:
             {
-                if (program_language == dmGraphics::ShaderDesc::LANGUAGE_SPIRV)
+                if (program_language == dmGraphics::ShaderDesc::LANGUAGE_SPIRV || program_language == dmGraphics::ShaderDesc::LANGUAGE_WGSL)
```

Comment:
For future reference (to myself), this should probably be put into a "context settings/properties/parameters" as an "NDC depth remap" setting or something